CoreScriptsRoot/Modules/PlayerlistModule.lua

```diff
@@ -1263,12 +1284,10 @@ local function createPlayerEntry(player, isTopStat)
 	local containerFrame, entryFrame = createEntryFrame(name, PlayerEntrySizeY, isTopStat)
 	entryFrame.Active = true
 
-	if not isTenFootInterface then
-		local function localEntrySelected()
-			onEntryFrameSelected(containerFrame, player)
-		end
-		entryFrame.MouseButton1Click:connect(localEntrySelected)
+	local function localEntrySelected()
```

Comment:
`localEntrySelected` isn't a very descriptive name for me. Also, if it is a callback to an event then we should start the name with `on`